main.tex

```diff
@@ -832,44 +869,37 @@
 
 \begin{enumerate}[label=(\alph*)]
 
-\item  drive in a safe and legal manner that observers all signs that are on Common Property in relation to the use of any vehicle;
+\item  drive or ride in a safe and legal manner that observers all signs that are on Common Property in relation to the use of any vehicle;
 
-\item  park wholly within their allocated car parking space;
+\item  park wholly within their allocated vehicle parking space(s);
```

Comment:
People are now allowed to park across 2 spaces, if they own both of them and they're next to each other.